modules/util/util.js

```diff
@@ -139,7 +139,7 @@ export function utilNoAuto(selection) {
   // https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion
 
   return selection
-    .attr('autocomplete', 'new-password')
+    .attr('autocomplete',  'off')
```

Comment:
Was there a good reason to change this?  The link above talks about `new-password` as a special value for this purpose.

https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion